docs/dev/logger.md

```diff
@@ -0,0 +1,52 @@
+### Logging
+
+Logging utilizes the [`github.com/cihub/seelog`](https://github.com/cihub/seelog) package as its underlying framework. 
+It is accessible via `pkg/util/log` and the `comp/core/log` component wrapper.
+Using the component wrapper is recommended, as it adheres to [component best practices](https://datadoghq.dev/datadog-agent/components/overview/).
+
+#### Writing a good log message
+
+In general, there are both a few rules and a few suggestions to follow when it comes to writing a
+(good) log message:
+
+- Messages must be written in English. No preference on which specific English dialect is used e.g.
+  American English, British English, Canadian English, etc.
+- Sentences must be capitalized, and end with a period.
+- Proper spelling and grammar when possible. Not all of us are native English speakers, and so this
+  is simply an ask, but not a hard requirement.
+- Identifiers, or passages of note, should be called out by some means i.e. wrapping them in
+  backticks or quotes.  Wrapping with special characters can be helpful in drawing the users eye to
+  anything of importance.
+- If it's longer than one or two sentences, it's probably better suited as a single sentence briefly
+  explaining the event, with a link to external documentation that explains further.
+
+#### Choosing the right log level
+
+Similarly, choosing the right level can be important, both from the perspective of making it easy
+for users to grok what they should pay attention to, but also to avoid the performance overhead of
+excess logging (even if we filter it out and it never makes it to the console).
```

Comment:
```suggestion
Choosing the right level can be important. Appropriate log levels make it easy for users to understand what they should pay attention to. They also avoid the performance overhead of
excess logging, even if the logs are filtered and never show on the console.
```

Reply:
`Choosing the right level can be important` -> `Choosing the right level is also very important`